nginx_proxy/config.yaml

```
@@ -26,16 +26,23 @@ options:
     active: false
     default: nginx_proxy_default*.conf
     servers: nginx_proxy/*.conf
+  proxyProtocol:
```

Multi-word configs in other add-ons usually use Python style naming with underlines, so:

```suggestion
  proxy_protocol:
```